### PR TITLE
Topical events About Edit page to GDS

### PR DESCRIPTION
**Pull request by farahTW**
This PR transits topical events about edit page to gds.
It does the following:

- Duplicated old edit and form files to legacy and updated controller with design system.
- Added new edit html for topical events about edit page.

**Screen shots:**

image

image

image
image

**No About page scenario:**
image

**Trello:**
https://trello.com/c/HjFFamct/152-edit-about-page

### CODE REVIEW

File: app/views/admin/topical_event_about_pages/_form.html.erb [hunk 1]
```diff
@@ -1,26 +1,54 @@
...
-
-      <%= form.save_or_cancel cancel: admin_topical_event_topical_event_about_pages_path(@topical_event) %>
-    <% end %>
-  </section>
-  <div class="col-md-4">
-    <%= legacy_simple_formatting_sidebar %>
+<%= form_with model: topical_event_about_page, url: admin_topical_event_topical_event_about_pages_path(topical_event_about_page.topical_event) do |form| %>
+  <%= render "govuk_publishing_components/components/input", {
+    label: {
+      text: "Name(required)",
```
Comment (davidtrussler) [inline app/views/admin/topical_event_about_pages/_form.html.erb:4]: We need a space before "required"

File: app/views/admin/topical_event_about_pages/_form.html.erb [hunk 2]
```diff
@@ -1,26 +1,54 @@
...
+      name: "topical_event_about_page[read_more_link_text]",
+      error_items: errors_for(topical_event_about_page.errors, :read_more_link_text)
+    },
+    maxlength: 255,
+    id: "topical_event_about_page_read_more_link_text",
+  } %>
+  <%= render "govuk_publishing_components/components/textarea", {
+    label: {
+      heading_size: "l",
+      text: "Summary(required)",
```
Comment (davidtrussler) [inline app/views/admin/topical_event_about_pages/_form.html.erb:29]: We need a space before "required"

File: app/views/admin/topical_event_about_pages/edit.html.erb [hunk 3]
```diff
@@ -1 +1,23 @@
-<%= render partial: 'form', locals: { page_title: "Edit page about #{@topical_event.name}", url: admin_topical_event_topical_event_about_pages_path(@topical_event, @topical_event_about_page) } %>
+<% content_for :page_title, "Edit #{@topical_event}" %>
+<% content_for :title, "Edit #{@topical_event}" %>
```
Comment (davidtrussler) [inline app/views/admin/topical_event_about_pages/edit.html.erb:2]: I think the `page_title` and `title` content need to be "Edit page about #{@topical_event}" (as in Bootstrap version)

File: app/views/admin/topical_event_about_pages/edit.html.erb [hunk 4]
```diff
@@ -1 +1,23 @@
-<%= render partial: 'form', locals: { page_title: "Edit page about #{@topical_event.name}", url: admin_topical_event_topical_event_about_pages_path(@topical_event, @topical_event_about_page) } %>
+<% content_for :page_title, "Edit page about #{@topical_event.name}" %>
+<% content_for :title, "Edit page about #{@topical_event.name}" %>
+<% content_for :title_margin_bottom, 6 %>
```
Comment (davidgisbey) [inline app/views/admin/topical_event_about_pages/edit.html.erb:3]: I think this should be 8 as Nik now want's these to be 50px.

**Review by davidtrussler — APPROVED** [review]
LGTM 👍

**Comment by nnagewad** [comment]
Looking good @farahTW . Just wondering if you have a screenshot when there is no About page data

**Comment by nnagewad** [comment]
Looks good as per screenshots @farahTW.

File: app/views/admin/topical_event_about_pages/edit.html.erb (hunk 4 above)
Comment (farahTW) [reply app/views/admin/topical_event_about_pages/edit.html.erb:3]: will change that

**Pull request merged**